pomodoro-cli.sh

```diff
@@ -10,28 +10,48 @@ function notify() {
   header=$1
   body=$2
   if [[ $kernel == "Darwin" ]]; then
-	osascript -e "display notification \"${body:?}\" with title \"${header:?}\""
+	osascript -e "display notification \"${body:?}\" with title \"${header:?}\" sound name \"~/.config/pomodoro-cli/sound.wav\""
```

Comment:
```suggestion
	osascript -e "display notification \"${body:?}\" with title \"${header:?}\""
```
It's the script without sound so no need for this part here.

Reply:
making this change